go/host/host.go

```diff
@@ -178,6 +178,18 @@ func (a *Node) MockedNewFork(b []common.EncodedBlock) {
 	a.forkRPCCh <- b
 }
 
+func (a *Node) SubmitAndBroadcastTx(encryptedParams common.EncryptedParamsSendRawTx) (common.EncryptedResponseSendRawTx, error) {
+	encryptedTx := common.EncryptedTx(encryptedParams)
+	encryptedResponse, err := a.EnclaveClient.SubmitTx(encryptedTx)
```

Comment:
Previously, we broadcast a tx around as soon as we received it. We now need to process it ourselves first, to be able to return the hash to the caller.